File: Heat/ViewModels/Customers/EditCustomerViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.Customers
    Public Class EditCustomerViewModel
        <Key> _
        Property ID As Integer

        <Required> _
        <Display(name:="Nome")> _
        Property Name As String
        <Display(name:="Indirizzo")> _
        Property Address As String
        <Display(name:="Città")> _
        Property City As String
        <Display(Name:="CAP")> _
        Property PostalCode As String
        <Display(name:="Provincia")> _
        Property District As String
        <Display(name:="Telefono 1")> _
        Property Telephone1 As String
        <Display(name:="Telefono 2")> _
        Property Telephone2 As String
        <Display(name:="Telefono 3")> _
        Property Telephone3 As String
        <Display(name:="Codice fiscale")> _
        Property Taxcode As String
        <Display(name:="Partita IVA")> _
        Property VAT_Number As String
        <Display(name:="IBAN")> _
        Property IBAN As String

        <EmailAddress> _
        Property EMail As String
        <Url> _
        Property Website As String

        <Display(name:="Abilitato")> _
        Property IsEnabled As Boolean
    End Class
End Namespace

File: Heat.ConvertedToC#/ViewModels/Customers/EditCustomerViewModel.cs
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Customers
{
    public class EditCustomerViewModel
	{
		[Key()]
		public int ID { get; set; }

		[Required()]
		[Display(Name = "Nome")]
		public string Name { get; set; }
		[Display(Name = "Indirizzo")]
		public string Address { get; set; }
		[Display(Name = "Città")]
		public string City { get; set; }
		[Display(Name = "CAP")]
		public string PostalCode { get; set; }
		[Display(Name = "Provincia")]
		public string District { get; set; }
		[Display(Name = "Telefono 1")]
		public string Telephone1 { get; set; }
		[Display(Name = "Telefono 2")]
		public string Telephone2 { get; set; }
		[Display(Name = "Telefono 3")]
		public string Telephone3 { get; set; }
		[Display(Name = "Codice fiscale")]
		public string Taxcode { get; set; }
		[Display(Name = "Partita IVA")]
		public string VAT_Number { get; set; }
		[Display(Name = "IBAN")]
		public string IBAN { get; set; }

		[EmailAddress()]
		public string EMail { get; set; }
		[Url()]
		public string Website { get; set; }

		[Display(Name = "Abilitato")]
		public bool IsEnabled { get; set; }

		[Display(Name = "Note")]
		[DataType(DataType.MultilineText)]
		public string Note { get; set; }

	}
}

